src/main/java/org/mule/extension/oauth2/internal/AbstractGrantType.java

```diff
@@ -166,6 +168,7 @@
   // the default token manager are recognized as equals even if
   // not initialized.
   private boolean defaultTokenManager;
+  private Boolean readsResponseBody;
```

Comment:
`boolean` with little `b`, to avoid conversion every time the getter is called

Reply:
Done!